test/powershell/Language/Classes/scripting.Classes.using.tests.ps1

```diff
@@ -523,6 +523,24 @@ using module FooForPaths
                 Pop-Location
             }
         }
+
+        It 'can be accessed by relative path with ./' {
+            $name = 'relative-forward-slash-paths'
+            'function Get-TestString { "Worked" }' | Set-Content "TestDrive:\modules\$name.psm1"
+
+            "using module ./$name.psm1; Get-TestString" | Set-Content "TestDrive:\modules\$name.ps1"
+
+            & "TestDrive:\modules\$name.ps1" | Should -BeExactly "Worked"
+        }
+
+        It 'can be accessed by relative path with .\' {
```

Comment:
Both new tests use identical code and we should merge it in one with using `-TestCases`.

Reply:
This should be done, but I'm not going to block on this.

Reply:
I can do this as soon as I get off work (about 5 PM EDT), best to change this to have best practices since it's such a simple change.